mod-linked-data/src/main/resources/karate-config.js

```diff
@@ -51,6 +57,10 @@ function fn() {
       for (var i = 0; i < 5; i++)
         text += possible.charAt(Math.floor(Math.random() * possible.length));
       return text;
+    },
+
+    sleep: function(seconds) {
```

Comment:
Can we update line 47 of `create-bib-srs.feature` to use this function?

Reply:
Sure! Done.